.github/workflows/ci.yml

```
@@ -109,8 +109,14 @@ jobs:
       - name: Install dependencies
         run: python -m poetry install
 
-      - name: Test with pytest
-        run: python -m poetry run pytest -v tests/
+      - name: Analyze with mypy
+        run: poetry run mypy .
```

Typing verification of code base without test is enough for us.

```suggestion
      - name: Analyze with mypy
        run: poetry run mypy happy_bank_core/
```